### PR TITLE
Manage automated testing in the background

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
Manage automated testing in the background

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/auto_test/tests/org/organization-manage.spec.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    await Promise.all([
+      page.waitForNavigation(/*{ url: 'https://erda.hkci.terminus.io/erda/orgCenter/safety?endAt=2021-10-28%2014%3A17%3A51&pageNo=1&startAt=2021-10-28%2013%3A17%3A51' }*/),
+      page.click('text=Audit log'),
+    ]);
+    const [page1] = await Promise.all([
+      page.waitForEvent('popup'),
+      page.waitForEvent('download'),
+      page.click('button:has-text("export")'),
+    ]);
+    await page1.close();
```
Comment (daskyrk) [inline shell/auto_test/tests/org/organization-manage.spec.ts:73]: There is no expect in this test case?

Comment (wuhuiluo) [reply shell/auto_test/tests/org/organization-manage.spec.ts]: modified

File: shell/auto_test/tests/org/organization-manage.spec.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    await page.click('textarea');
+    await page.fill('textarea', title);
+    await page.click('button:has-text("save")');
+    await page.click('[placeholder="search by project name"]');
+    await page.fill('[placeholder="search by project name"]', title);
+    await expectExist(`text=${title}`);
+    await page.click('.ant-table-row td:nth-child(8)');
+  });
+
+  test('Market-administration', async ({ page, expectExist, goTo }) => {
```
Comment (daskyrk) [inline shell/auto_test/tests/org/organization-manage.spec.ts:48]: Where is `Market-administration`?

File: shell/auto_test/tests/org/organization-manage.spec.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+import { Role, test, expect } from '../../fixtures';
+import Base from '../pages/base';
+const title = 'terminus' + Date.now();
+const testData = {
+  image: 'app/images/Erda.png',
+};
+
+Role('Manager', () => {
+  test('management-center', async ({ page, expectExist, goTo }) => {
```
Comment (daskyrk) [inline shell/auto_test/tests/org/organization-manage.spec.ts:22]: Test case `management-center` is just create project ?

File: shell/auto_test/tests/org/organization-manage.spec.ts (hunk 2 above)
Comment (wuhuiluo) [reply shell/auto_test/tests/org/organization-manage.spec.ts]: modified

File: shell/auto_test/tests/org/organization-manage.spec.ts (hunk 3 above)
Comment (wuhuiluo) [reply shell/auto_test/tests/org/organization-manage.spec.ts:24]: modified, Set the rollback point, click the project to jump to the project market

**Pull request merged**